frontend/src/views/web/challenge/leaderboard.html

```diff
@@ -11,6 +11,16 @@
                         <div></div>
                     </div>
                 </div>
+                <div class="ev-md-container ev-panel-title" ng-if="challenge.page.leaderboard_description">
+                    <div class="row">
+                        <div class="col xs12 s12">
+                            <strong>Description</strong>
+                        </div>
+                        <div class="col s12">
+                            {{challenge.page.leaderboard_description}}
```

Comment:
Ohh..just noticed it. Can you please use `ng-bind-html` in order to render HTML from the backend.

Reply:
Thanks for pointing it out : )